cosi.proto

```diff
@@ -182,19 +182,15 @@ message ProvisionerCreateBucketRequest {
 }
 
 message ProvisionerCreateBucketResponse {
-    // Intentionally left blank
+    // BucketName returned here is expected to be the globally unique 
+	// identifier for the bucket in the object storage provider 
+	string BucketName = 1;
 }
 
 message ProvisionerDeleteBucketRequest {
     // This field is REQUIRED
     // Protocol specific information required by the call is passed in as key,value pairs.
     Protocol protocol = 1;
-
```

Comment:
don't we need the bucket id? How does the provisoner know which bucket to delete?